### ISSUE TITLE
LEXICON-GEARGGUS and others (

### ISSUE THREAD

**Issue by albbas** [giellalt/lang-sme#137]
This issue was created automatically with bugzilla2github
# Bugzilla Bug 50
Date: 2005-02-20T14:01:12+01:00
From: Thomas Omma &lt;<thomas.omma>&gt;
To:   Maaren Palismaa &lt;<Maren.Palismaa>&gt;
CC:   trond.trosterud

Depends on:   #56
Last updated: 2007-09-27T09:55:51+02:00

**Comment by albbas** [comment]
## Comment 149
Date: 2005-02-20 14:01:12 +0100
From: Thomas Omma &lt;<thomas.omma>&gt;

When adding the standard (western) form eabbo to LEXICON EABBO/EAMOS_CONT a new bug arised. 
The form for Comp+Sg+Nom/Gen/Acc/Ill+PxSg1/PxSg2/PxDu1/PxDu2/PxPl1/PxPl2 should be 
gergosebbon etc. and the right form do get generated and have always so. The problem is that the 
wrong form gergoseabbon etc. too gets generated after the addition to LEXICON EABBO/EAMOS_CONT 
has been done. Now, I can't understand how these forms too can be generated since the addition 
(eabbo) points to LEXICON EABBU exactly like the poInter from the substandard form eabbu and there 
hasn't been any changes done in LEXICON EABBU in this connection.

**Comment by albbas** [comment]
## Comment 189
Date: 2005-03-07 10:33:20 +0100
From: Thomas Omma &lt;<thomas.omma>&gt;

This problem with lack of diftongsimplification isn't apparently just in the Eabbo-comparatives. I found 
it in word 'miel

**Comment by albbas** [comment]
## Comment 197
Date: 2005-03-09 15:04:39 +0100
From: Thomas Omma &lt;<thomas.omma>&gt;

Same problem found in nouns:

heaibu+N+Sg+Ill
heaibui                                          should be: heibui
heibui

heaibu+N+Sg+Loc
heaibus
heaibbus

heaibu+N+Sg+Com
heaibuin                               should be: heibuin
heaibbuin

heaibu+N+Ess
heaibun

heaibu+N+Pl+Nom
heaibut
heaibbut

heaibu+N+Pl+Gen
heaibuid                                  should be: heibuid
heaibbuid

heaibu+N+Pl+Acc
heaibuid                                  should be: heibuid
heaibbuid

heaibu+N+Pl+Ill
heaibuide                                 should be: heibuide
heaibbuide

heaibu+N+Pl+Loc
heaibuin                                 should be: heibuin
heaibbuin

heaibu+N+Pl+Com
heaibuiguin                           should be: heibuiguin
heaibbuiguin

**Comment by albbas** [comment]
## Comment 212
Date: 2005-03-17 10:34:28 +0100
From: Thomas Omma &lt;<thomas.omma>&gt;

This is right, please ignore the paradigm above


                                                              should be:

miel

**Comment by albbas** [comment]
## Comment 218
Date: 2005-03-29 09:49:45 +0200
From: Thomas Omma &lt;<thomas.omma>&gt;

The two bugs with miel

**Comment by albbas** [comment]
## Comment 270
Date: 2005-05-25 15:06:40 +0200
From: Trond Trosterud &lt;<trond.trosterud>&gt;

OK, ok, this is abug, and I will look at it, but I await a decision from the Normative Authority (Maaren, that 
is) on to what extent we should accept dialects form (i.e., non Guovdageaidnu forms).

**Comment by albbas** [comment]
## Comment 274
Date: 2005-05-26 08:47:12 +0200
From: Thomas Omma &lt;<thomas.omma>&gt;

Yeah. Note that it is the adding of the standard forms that has created this bug.

**Comment by albbas** [comment]
## Comment 971
Date: 2006-05-24 15:28:14 +0200
From: Thomas Omma &lt;<thomas.omma>&gt;

This is also a G2/G3 issue. Returning to it with #56.

**Comment by albbas** [comment]
## Comment 1957
Date: 2007-09-27 09:55:51 +0200
From: Trond Trosterud &lt;<trond.trosterud>&gt;

Bug closed, sloppy wording in the initial comment, and the analysis
gergoseabbon    gearggus+A+Comp+Ess

gergosebbon
gergosebbon     gearggus+A+Comp+Sg+Acc+PxSg1
gergosebbon     gearggus+A+Comp+Sg+Gen+PxSg1
gergosebbon     gearggus+A+Comp+Sg+Nom+PxSg1
is what we want.
Another issue is that the last analysis should perhaps not be part of the speller.
